src/utils.ts

```diff
@@ -73,16 +73,16 @@ export async function sha256(string) {
  */
 export const createEmptyDataFrame = (
   propNames: string[],
-  timeField: string,
   isLogsRequest: boolean,
   targetType: QueryType,
   logMessageField?: string,
-  logLevelField?: string
+  logLevelField?: string,
+  timeField?: string
 ): MutableDataFrame => {
   const series = new MutableDataFrame({ fields: [] });
 
   //PPL table response should add time field only when it is part of the query response
-  if (targetType === QueryType.Lucene || isLogsRequest) {
+  if (timeField && (targetType === QueryType.Lucene || isLogsRequest)) {
```

Comment:
The function was adding a random "Field 1" column to traceDetails, so i did this quick fix to not add it. All older function calls have a time field, so this shouldn't change anything. I thought about removing the rest of the conditions, but this function is being called from places in the app that (I think) still need these conditions.